crates/valence_client/src/lib.rs

```diff
@@ -960,6 +1006,62 @@ fn read_data_in_old_view(
     );
 }
 
+fn update_layer_view(
+    mut clients: Query<(
+        &mut Client,
+        &mut EntityRemoveBuf,
+        &OldPosition,
+        &OldViewDistance,
+        Option<&ClientLayerSet>,
+    )>,
+    entities: Query<(EntityInitQuery, &OldPosition, Option<&Layer>)>,
```

Comment:
Why do we need to query for `Option<&Layer>` and `Option<&ClientLayerSet>` if we don't handle the `None` variant?